presto-hive/src/test/java/io/prestosql/plugin/hive/AbstractTestHiveFileSystem.java

```diff
@@ -245,7 +245,7 @@ public void testGetRecords()
             ConnectorSplitSource splitSource = splitManager.getSplits(transaction.getTransactionHandle(), session, table, UNGROUPED_SCHEDULING);
 
             List<ConnectorSplit> splits = getAllSplits(splitSource);
-            assertEquals(splits.size(), 1);
```

Comment:
When 43bd519052ba4c56ff1f4fc807075637ab5f4f10 was being worked on, there was no assertion here.
It was added later in a389636715d2b7d2fdfd9d3a5fe113f175926c3c. @electrum do you remember where did `1` come from?